spec/features/early_career_payments_claim_spec.rb

```diff
@@ -76,13 +76,16 @@
 
     # - In which academic year did you start your undergraduate ITT
     expect(page).to have_text(I18n.t("early_career_payments.questions.itt_academic_year.qualification.#{claim.eligibility.qualification}"))
-    expect(page).to have_text("Did you do a deferred or part time undergraduate ITT")
     expect(page).to have_text("2017 to 2018")
     expect(page).to have_text("2018 to 2019")
     expect(page).to have_text("2019 to 2020")
     expect(page).to have_text("2020 to 2021")
     expect(page).to have_text("2021 to 2022")
 
+    expect(page).to have_text("Did you do a deferred or part time undergraduate ITT")
+    find("span", text: "Did you do a deferred or part time undergraduate ITT").click
```

Comment:
I suspect you'll have a good reason for this, but why isn't it just `click_on "Did you do a deferred or part time undergraduate ITT"`?

Reply:
It's because `click_on` works only for links and buttons, and that element is a span